src/extensions/default/AutoUpdate/main.js

```diff
@@ -46,7 +46,8 @@ define(function (require, exports, module) {
     var _modulePath = FileUtils.getNativeModuleDirectoryPath(module),
         _nodePath = "node/AutoUpdateDomain",
         _domainPath = [_modulePath, _nodePath].join("/"),
-        updateDomain;
+        updateDomain,
```

Comment:
These changes should not be part of this PR, as they are already present in [14320](https://github.com/adobe/brackets/pull/14320/files)